packages/babel-preset-react-app/index.js

```diff
@@ -134,3 +136,29 @@ if (env === 'test') {
     // ]);
   }
 }
+
+module.exports = function(context, opts = {}) {
+  const moduleTypes = ['commonjs', 'amd', 'umd', 'systemjs'];
+  let modules = 'commonjs';
+
+  if (opts !== undefined) {
+    if (opts.modules !== undefined) modules = opts.modules;
+  }
+
+  if (modules === true) {
+    modules = 'commonjs';
+  }
+
+  if (modules !== false && moduleTypes.indexOf(modules) === -1) {
+    throw new Error(
+      "Preset es2015 'modules' option must be 'false' to indicate no modules\n" +
```

Comment:
Seems weird to mention `preset-es2015` when the preset uses `preset-env`... probably not even necessary to mention either at all

Reply:
I simply copied the error message form [babel-preset-es2015 source](https://github.com/babel/babel/blob/26e4911eb22a75f6581febe0cedc245a683c55cb/packages/babel-preset-es2015/src/index.js#L47-L48).

But I agree that mentioning the specific package isn't required.